docker-build-images/golang-build.Dockerfile

```diff
@@ -1,38 +1,18 @@
-# SL6.5 has the oldest version of glibc that we support.  Build against
-# that.
-FROM ringo/scientific:6.5
+FROM golang:1.7
 
 MAINTAINER Shaun Crampton <shaun@tigera.io>
 
-# gcc for cgo
-RUN yum install -y \
-		g++ \
-		gcc \
-		libc6-dev \
-		make \
-		curl \
-		tar \
-		wget
+RUN apt-get update && \
+    apt-get install -y bsdmainutils && \
```

Comment:
What's `bsdmainutils` and why is it required? Can you add a brief comment to explain?